src/witan/cic/tuairisc/config.clj

```diff
@@ -0,0 +1,27 @@
+(ns witan.cic.tuairisc.config
```

Comment:
config? why did you call it this? It's not bad, I'm just not sure I understand your line of thinking?

Reply:
this is about parsing the config files and pulling out interesting things.

Reply:
Suggestion: I do wonder whether config-parser or similar would be a better name then, as this suggests it associated with a config for something.